package.json

```diff
@@ -65,7 +65,9 @@
     "react-native-screens": "3.18.0",
     "react-native-web": "0.18.9",
     "sentry-expo": "6.0.0",
-    "yup": "0.32.11"
+    "yup": "0.32.11",
+    "expo-image-manipulator": "11.0.0",
+    "expo-file-system": "15.1.1"
```

Comment:
this is needed, as imagepicker only retrieves file sizes for iOs images, but not for android